events/service/src/main/java/org/projectnessie/events/service/util/SubscriberUtil.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (C) 2023 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.projectnessie.events.service.util;
+
+import org.projectnessie.events.api.CommitEvent;
+import org.projectnessie.events.api.ContentRemovedEvent;
+import org.projectnessie.events.api.ContentStoredEvent;
+import org.projectnessie.events.api.Event;
+import org.projectnessie.events.api.MergeEvent;
+import org.projectnessie.events.api.ReferenceCreatedEvent;
+import org.projectnessie.events.api.ReferenceDeletedEvent;
+import org.projectnessie.events.api.ReferenceUpdatedEvent;
+import org.projectnessie.events.api.TransplantEvent;
+import org.projectnessie.events.spi.EventSubscriber;
+
+public class SubscriberUtil {
+
+  public static void notifySubscriber(EventSubscriber subscriber, Event event) {
```

Comment:
Sorry for re-opening this one...
Wouldn't it be easier to have this method in `EventSubscriber`?

Reply:
OK but we could even have it on `EventType` directly:

```java
EventType eventType = ...
eventType.notifySubscriber(subscriber, event);
```

This would avoid the giant `switch` block. Wdyt?

Reply:
Nah scratch that, these are in separate modules now (they were together initially). I will do as you suggest.